contrib/python/pypodman/pypodman/lib/report.py

```diff
@@ -26,18 +41,24 @@ def __init__(self, columns, heading=True, epilog=None, file=sys.stdout):
         """
         self._columns = columns
         self._file = file
+        self._format_string = None
+        self._formatter = ReportFormatter()
         self._heading = heading
         self.epilog = epilog
-        self._format = None
 
     def row(self, **fields):
         """Print row for report."""
         if self._heading:
             hdrs = {k: v.display for (k, v) in self._columns.items()}
-            print(self._format.format(**hdrs), flush=True, file=self._file)
+            print(
+                self._formatter.format(self._format_string, **hdrs),
```

Comment:
Ugg, if I see the word 'format' one more time, I'm gunna....

Reply:
I know, but the base class is Formatter with the method format().  Making up other names seemed more confusing.